packages/appstore/src/components/modals/cashier-wallet-modal/cashier-wallet-modal.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Modal } from '@deriv/components';
+import CashierWalletModalHeader from './cashier-wallet-modal-header';
+import CashierWalletModalBody from './cashier-wallet-modal-body';
+import { observer, useStore } from '@deriv/stores';
+
+const CashierWalletModal = observer(() => {
```

Comment:
```suggestion
import WalletModalHeader from './wallet-modal-header';
import WalletModalBody from './wallet-modal-body';
import { observer, useStore } from '@deriv/stores';

const WalletModal = observer(() => {
```

I think `WalletModal` would do instead of `CashierWalletModal`.